spec/ascii.spec.js

```diff
@@ -0,0 +1,41 @@
+/**
+ *  File: ASCII.spec.js
+ *  Author(s): unguest
+ */
+
+require('amd-loader');
+// for tests that need RequireJS
+define(function(require, exports, module) {
+  'use strict';
+
+  describe('src/ASCII.js =>', ()=>{
+    const encodeToASCII = require('../src/convertors/ascii').encodeToASCII;
+    it('Should encode and decode pure text', () => {
+      const clearText = 'javascript';
+      // eslint-disable-next-line max-len
+      const encodedText = '106971189711599114105112116';
```

Comment:
I was pretty skeptical about ASCII convertor because of it's ambiguity. Look at the encodedText here. `10697...` What does this signify? 10, 69, 7.... or 106, 97....?
This will not be ambiguous for strings having only characters, but when we're including special characters, decoding might cause ambiguity. So, we might need to introduce some kind of separator after each character.

Reply:
Also, there are some linting errors causing the Travis build to fail.